site/hxapi/transfers/av/index.md

```diff
@@ -0,0 +1,503 @@
+| Infants     | Integer   | [0-9]            | N          | Number of Infants requiring a transfer.                                                                               |
+| OutFlight   | String    | [A-Z0-9]         | Y*         | A flight number is required if the journey involves being picked up or dropped off at an Airport.                     |
+| ReturnFlight| String    | [A-Z0-9]         | N*         | A flight number is required if the return journey involves being picked up or dropped off at an Airport.              |
+
+#### Dates and Times
+It's important to note that availability should be requested with accurate dates and times. A good scenario would be if an Airport Transfer is required, where the `PickUp` would be an Airport and the `DropOff` would be a Hotel. The `FromDate` and `FromTime` should be the flight arrival date and time. If a return transfer is requested, then the `ReturnDate` and `ReturnTime` should be the flight departure date and time, our system will calculate an estimated pickup time from the Hotel.
+
+## Availability Response
+
+If availability has been found for the requested parameters, the response will contain a `Transfers` field with all the nested Transfers available. A summary of the request received can be found in the header field at the end of the response.
+
+For a detailed explanation of the fields returned, please see below:
+
+| Field                      | Additional Information                                                                                      |
+|----------------------------|-------------------------------------------------------------------------------------------------------------|
+| Transfers/Name             | The name of the Transfer.                                                                                   |
+| Transfers/Code             | The product code for the transfer, which must be used in the booking request.                               |
+| Transfers/BookingURL       | The URL to POST the booking request to for this particular product.                                         |
+| Transfers/MoreInfoURL      | N/A for Transfers.                                                                                          |
+| Transfers/TotalPrice       | The total price of the Transfer/s.                                                                          |
+| Transfers/NonDiscPrice     | If a discount has been applied, this is the total price before discount.                                    |
+| Transfers/SalesCurrency    | Three character identifier for the sale currency to qualify the price.                                      |
+| Transfers/Images/Src       | Link to a image for the product. Prefix with `https:`.                                                      |
```

Comment:
Any way we can just add this prefix? Might be a pain, but seems weird to get them to add something when there is only one choice anyway....

Reply:
Yeah, it's a good point. I saw the other products are set up like this. So I was thinking if the partner has existing logic to prefix the image URL, I'll keep it the same.

Reply:
👍